spacelift/data_azure_integrations_test.go

```diff
@@ -0,0 +1,86 @@
+package spacelift
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal/structs"
+	. "github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal/testhelpers"
+)
+
+func TestAzureIntegrationsData(t *testing.T) {
+	t.Run("when looking up integrations", func(t *testing.T) {
+		subId1 := acctest.RandStringFromCharSet(5, acctest.CharSetAlpha)
+		subId2 := acctest.RandStringFromCharSet(5, acctest.CharSetAlpha)
+		first := &structs.AzureIntegration{
+			AdminConsentURL:       "https://org.org",
```

Comment:
I think the `AdminConsentURL` is read-only, so I don't think there's much point setting it.